config.example.json

```diff
@@ -1,9 +1,9 @@
 {
   "host": "http://localhost:8080",
   "username": "admin",
-  "password": "admin",
-  "shareUser": "admin",
+  "password": "password",
```

Comment:
This is by now less of an example and more a strictly required config thx to the new Dockerfile. Meaning maybe better to remove the example file and check in the real thing?

Reply:
Yeah I thought of that. It would make the config file redundant so we could hardcode also it into the tests.
In the end that would mean tests can only run using the development instance which sounds like a good idea to me, because everything would get reproducible.